src/functionalTest/resources/scenarios/RIA-7412-review-interpreter-language-flag.json

```diff
@@ -0,0 +1,160 @@
+{
+  "description": "RIA-7412 review language requirements for legal rep",
+  "enabled": "true",
```

Comment:
Should this test be enabled by $featureFlag.isSubmitHearingRequirementsEnabled ?